crates/astria-sequencer/src/app/tests_app/mod.rs

```diff
@@ -361,10 +358,6 @@ async fn app_create_sequencer_block_with_sequenced_data_and_deposits() {
         .unwrap();
     app.commit(storage).await;
 
-    // ensure deposits are cleared at the end of the block
```

Comment:
ensure the deposit nonce is cleared still?

Reply:
Done in [639449c](https://github.com/astriaorg/astria/pull/1525/commits/639449cb5f751222f20548f4ee3d1acc18698814).

I've made `get_deposit_nonce` private to `bridge::state_ext` since it's not needed anywhere outside of there, so for this test case I just created a new test-only asserting function.